src/customize-pop/applications-menu.md

```diff
@@ -0,0 +1,31 @@
+# Applications Settings
+
+Options to control application access to system features and resources are located in the `Settings` ➞ `Applications` tab.
+
+![Application Settings](/images/application-settings/application-settings.png)
+
+## Built-in Permissions
+
+View the data and system services requested or required by an application. Permissions can include access to system devices, using the system's network connection, reading and writing to the file system, and the ability to change settings. Many applications include built-in permissions that cannot be modified by the user. View permissions granted to an application by clicking the `Built-in Permissions` button.
+
+![Built-in Permissions](/images/application-settings/built-in-permissions.png)
+
+## Integration
+
+Disable or enable system-wide features used by an application. These features include the ability to appear in search, display notifications, and run in the background.
+
+## Default Handlers
+
+Configure specific file MIME types and links handled by the application. Click an entry to display more information about the file types assigned to the application.
+
+![Default Handlers](/images/application-settings/default-handlers.png)
+
+Clicking `Unset` will remove the file type association with the application. These associations can be restored using the [File Manager](default-applications.md#set-default-applications-using-the-file-manager) or a [command-line procedure](default-applications.md#modify-the-mimeappslist-file).
+
+To set a default application for a large number of similar file types, such as images, see the [Default Applications](default-applications.md) section.
+
+## Usage
+
+Display an application's disk usage. Click on the `Storage` button to view space used by the application itself, data created by the application, and temporary data cached by the application. Click `Clear Cache` to clear an application's locally stored data.
```

Comment:
> Click `Clear Cache` to clear an application's locally stored data.

This almost sounds like it would clear the app's non-cache data to. Is that the case? (Something like `an application's temporary data` might be an alternative way to phrase cache, if necessary.)

Reply:
Clarified